testingii/src/App.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import './App.css';
+import Display from './display/Display.js'
+import Dashboard from './dashboard/Dashboard.js'
+
+class App extends Component {
+  state = {
+    balls: 0,
+    strikes: 0,
+    outs: 0 
+  }
+
```

Comment:
Okay, so this type of nesting isn't necessarily my favorite way to handle this type of functionality because it can easily get very convoluted and difficult to follow your own logic, let alone have another developer go behind you and have to unravel it--which incidentally I just did.  You are better off abstracting smaller bits of functionality (i.e. create a function that clears out state and use that instead of clearing state each time you need to... etc).